backend/src/api/group/content-types/group/schema.json

```diff
@@ -31,7 +31,7 @@
           "localized": true
         }
       },
-      "type": "string",
+      "type": "text",
```

Comment:
In my opinion, this should stay as string

Reply:
Fixed it, thank you!